### DISCUSSION TITLE
Revalidating Cache (Tag|Page) from External Application

### DISCUSSION THREAD

**Discussion by victorcesae** [vercel/next.js#63557] [App Router]
I have an external backend application that when i create an order, this order is updated in the backend and i have to send the new information to my frontend, but if i use next.js caching in this pages i can't `revalidateTag` or `revalidatePage` from the external backend, because accordingly with the next.js 14 [docs](https://nextjs.org/docs/app/building-your-application/caching#revalidatepath) the data will only be revalidate if the next page accessed is the page or one page with the cache revalidated.

Anyone has some work arround to this situation?

**Comment by icyJoseph** [comment]
Maybe do polling with a server action? Have the server action check if the order is updated, and if so, revalidate the path.

**Reply by victorcesae** [reply]
I'm in now days using CSR to render the page so, i'm working with socket.io and send the order changes to the user and update the order info with tanstack/react-query, but i don't see how with server action will work
